Modules/Module_Processing/Module_Population/xASL_stat_GetROIstatistics.m

```diff
@@ -392,6 +407,11 @@
 						for iMask=1:size(x.S.InputMasks,3)
 							if sum(SumList(iROI,iMask))~=0 % skip empty ROIs
 								x.S.InputMasks(:,iROI,iMask) = xASL_im_CreatePVEcROI(x,x.S.InputMasks(:,iROI,iMask), pGM_MNI, pWM_MNI);
+                            else
+                                fprintf('\n');
+                                numVoxels = sum(x.S.InputMasks(:,iROI,iMask));

```

Comment:
same as above

Reply:
Done in 46c421bcd1aed8156cbe146a8cf708930de3d337